ETL_csv_to_python.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import numpy as np
+from datetime import datetime as dt
+
+data_ip = pd.read_csv('car_prices.csv')
```

Comment:
Just a placeholder, if you can use functions for extracting , transforming and loading separately